### ISSUE TITLE
[BUG] ERR_CONNECTION_REFUSED // CORS FAILURE

### ISSUE THREAD

**Issue by Supermarcel10** [Mastermindzh/tidal-hifi#486]
### Information
When attempting to start playback, the progress bar proceeds, but no sound is outputted. When the progress bar reaches aprox 00:13, it resets to 00:00 and an S1002 error popup is shown.

It is possible that this may be an issue with Tidal's backend configuration, and out of control.

### Steps to Reproduce
1) Open tidal-hifi
2) Open Developer Console (Ctrl + Shift + I)
3) Move to Console or Network tab
4) Start playback
5) Network issues can be seen

### Exception
![image](https://github.com/user-attachments/assets/57662916-73d9-48e2-ae2a-1377b770b1aa)
![image](https://github.com/user-attachments/assets/ff560365-38df-4b18-9548-94a37a1e9ffa)

### Setup
Installed via [nix configuration](https://github.com/Supermarcel10/NixOSConfig/blob/829734183f135bf65d42494dd6998b75567d52c8/configuration.nix#L123) via nixpkgs.
```nix
environment.systemPackages = with pkgs; [
     # ...
     tidal-hifi
     # ...
];
```

Tested on versions:
- `5.12.0` (Latest stable on NixPkgs 24.05)
- `5.16.0` (Latest on NixPkgs Unstable)

**Comment by Mastermindzh** [comment]
This is the exact thing that happens on Windows.
Can you try removing tidal-hifi's config directory to give it a jolt?

On most systems it's located in the regular config folder (~/.config/tidal-hifi)
That would require you to relog as well btw.

**Comment by Supermarcel10** [comment]
From testing, I have tried removing the `~/.config/tidal-hifi` directory and it has not changed anything from my testing. 

I was able to resolve the issues by disconnecting and connecting my network again - even on a wired connection. Changing network configurations or such did not make any differences. So I have no idea why that would resolve the issue. Maybe something with caches, or DNS caches might be the culprit, but I have yet to investigate further what would be the primary cause.
